packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -196,21 +194,15 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
     }
   }, [resetUploadingState])
 
-  useEffect(() => {
-    if (uploadActionTrigger) {
-      const uploadActionPromise = dispatchThunk(uploadFile())
-
-      //abort on dismount or when a new upload has been triggered
-      return () => {
-        if ("abort" in uploadActionPromise) {
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-explicit-any
-          ;(uploadActionPromise as any).abort()
-        }
-      }
+  const onFileInputChange = () => {
```

Comment:
You could also remove useRef and put argument to this callback method: `e: ChangeEvent<HTMLInputElement>`. 
But maybe there is a benefit of using useRef that i don't see here.